src/nft/pages/collection/index.tsx

```diff
@@ -27,7 +27,8 @@ import { ThemedText } from 'theme'
 
 const FILTER_WIDTH = 332
 const BAG_WIDTH = 324
-export const COLLECTION_BANNER_HEIGHT = 288
+const BannerWrapper = styled(Box)``
```

Comment:
Why not use Box directly? Seems a bit redundant to have an empty wrapper like this?

Reply:
Also, just nitpicking, but I think it would be great to group constants together (including the one on line 31) and separate from this component